src/state/shellscripts/bwrap.sh

```diff
@@ -11,6 +11,13 @@ if ! command -v bwrap >/dev/null; then
     exit 10
 fi
 
+OPAMVERSION=`opam --version`
+if [ "$OPAMVERSION" != "$SCRIPTOPAMVERSION" ]; then
```

Comment:
Wouldn't it be better to use a key which itself isn't a shell variable? Like `@SCRIPTOPAMVERSION@` or something?